app/views/extra_registrations/_name.html.slim

```diff
@@ -1,3 +1,5 @@
-= f.govuk_fieldset legend: { text: "Name", tag: :h1, size: 'l' } do
-  = f.govuk_text_field :first_name, width: 'one-half', label: { text: t(:first_name) }
-  = f.govuk_text_field :last_name, width: 'one-half', label: { text: t(:last_name) }
+= f.govuk_fieldset legend: { text: t(:heading, scope: [:extra_registration, current_step]), tag: :h1, size: 'l' } do
```

Comment:
@ebrett can we DRY up the views by configuring the defaults like tag and size in the initialiser.

Reply:
I did not see an option in the config documentation, unless I'm missing something: https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder

Reply:
Is it possible to set defaults in `config/initializers/govuk_formbuilder`

Reply:
```ruby
{
default_legend_size: 'l',
default_legend_tag: :h1,
}
```

Reply:
https://www.rubydoc.info/gems/govuk_design_system_formbuilder/0.9.4/GOVUKDesignSystemFormBuilder/Containers/Fieldset if fieldsets behave differently then perhaps we can just drop the tag as `h1` is the default.

Reply:
If that can be done - it might be worth doing the same for width (as 'two-thirds') on some elements